packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,48 +247,183 @@ function ListView() {
+                      onValueChange={value => handleCheckboxChange('main', value)}
+                    />
+                  </Th>
+                  <Th>
+                    <TableLabel>name</TableLabel>
+                  </Th>
+                  <Th>
+                    <TableLabel>url</TableLabel>
+                  </Th>
+                  <Th width="30%">
+                    <TableLabel>status</TableLabel>
+                  </Th>
+                  <Th>
+                    <VisuallyHidden>actions</VisuallyHidden>
+                  </Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {webhooks.map(webhook => (
+                  <Tr key={webhook.id}>
+                    <Td>
+                      <BaseCheckbox
+                        aria-label={`Select ${webhook.name}`}
+                        value={webhooksToDelete?.includes(webhook.id)}
+                        onValueChange={value =>
+                          handleCheckboxChange('secondary', value, webhook.id)}
+                        id={`Select ${webhook.id}`}
+                        name={`Select ${webhook.name}`}
+                      />
+                    </Td>
+                    <Td>
+                      <Text highlighted textColor="neutral800">
+                        {webhook.name}
+                      </Text>
+                    </Td>
+                    <Td>
+                      <Text textColor="neutral800">{webhook.url}</Text>
+                    </Td>
+                    <Td>
+                      <Row>
+                        <Switch
+                          onLabel="Enabled"
+                          offLabel="Not enabled"
+                          label={`${webhook.name} status`}
+                          selected={webhook.isEnabled}
+                          onChange={() => handleEnabledChange(!webhook.isEnabled, webhook.id)}
+                          visibleLabels
+                        />
+                      </Row>
+                    </Td>
+                    <Td>
+                      <Row justifyContent="flex-end">
+                        {canUpdate && (
+                          <IconButton
+                            onClick={() => {
+                              handleGoTo(webhook.id);
+                            }}
+                            label="Edit"
+                            icon={<EditIcon />}
+                            noBorder
+                          />
+                        )}
+                        {canDelete && (
+                          <Box paddingLeft={1}>
+                            <IconButton
+                              onClick={() => handleDeleteClick(webhook.id)}
+                              label="Delete"
+                              icon={<DeleteIcon />}
+                              noBorder
+                            />
+                          </Box>
+                        )}
+                      </Row>
```

Comment:
Instead of doing this, in v9 (or 10) of parts, you can use:

```jsx
<Stack horizontal size={1}>
 <Button1/>
 <Button2/>
</Stack>
```

and it will do the same as what you're trying to do 😊